pages/about.js

```diff
@@ -57,15 +57,11 @@ If you need help organising drinks, cables, put out chairs, etc, just let the or
 Signage helps people find your building, floor or office if you aren't located somewhere completely obvious. We encourage you to make use of our logo and put up signage in relevant places to give our attendees some breadcrumbs to follow.
 
 Download our logo here: [PNG](/static/images/cphjs.png)
-
-### How do I contact you?
```

Comment:
This section should still be here and with both the email and twitter account :)

Reply:
added back